params/defaults.go

```diff
@@ -39,6 +39,10 @@ const (
 	// allow us avoid syncing node.
 	RinkebyEthereumNetworkURL = "https://rinkeby.infura.io/nKmXgiFgc2KqtoQ8BCGJ"
 
+	// GoerliEthereumNetworkURL is an open RPC endpoint to Goerli network
+	// Other RPC endpoints are available here: http://goerli.blockscout.com/
+	GoerliEthereumNetworkURL = "http://goerli.blockscout.com/"
```

Comment:
So Infura does not support Goerli?

Reply:
yeah, no rpc endpoint from them. they are going to setup one in the future. also they run one of the block signers for this poa network